scalecodec/type_registry/polymesh.json

```diff
@@ -1443,6 +1464,52 @@
         ["running_count", "u32"]
       ]
     },
+    "GranularCanTransferResult": {
```

Comment:
I think these types are RPC types so we may don't need it as explorer doesn't know about the RPC queries

Reply:
I think it's worth having just so that this file stays consistant with `polymesh_schema.json` but I'm open to removing it if you don't agree